mathesar_ui/src/stores/abstract-types/operations/filtering.ts

```diff
@@ -97,32 +97,46 @@ const filterResponse: AbstractTypeFilterDefinitionResponse[] = [
   },
   {
     id: 'json_array_length_greater_than',
-    name: 'number of elements is greater than',
+    name: 'number of elements >',
     uiTypeParameterMap: {
       [abstractTypeCategory.Array]: [abstractTypeCategory.Number],
     },
   },
   {
     id: 'json_array_length_greater_or_equal',
-    name: 'number of elements is greater than or equal to',
+    name: 'number of elements >=',
     uiTypeParameterMap: {
       [abstractTypeCategory.Array]: [abstractTypeCategory.Number],
     },
   },
   {
     id: 'json_array_length_less_than',
-    name: 'number of elements is less than',
+    name: 'number of elements <',
     uiTypeParameterMap: {
       [abstractTypeCategory.Array]: [abstractTypeCategory.Number],
     },
   },
   {
     id: 'json_array_length_less_or_equal',
-    name: 'number of elements is less than or equal to',
+    name: 'number of elements <=',
     uiTypeParameterMap: {
       [abstractTypeCategory.Array]: [abstractTypeCategory.Number],
     },
   },
+  {
+    id: 'json_array_not_empty',
+    name: 'array is not empty',
+    uiTypeParameterMap: {
+      [abstractTypeCategory.Array]: [abstractTypeCategory.Number],
```

Comment:
For this filter, we do not expect any user input. So, this should be:
```suggestion
      [abstractTypeCategory.Array]: [],
```

Reply:
Thanks for your suggestions, Pavish! I have made that change.